tests/api/v1/verbs/doPost.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/api/v1/verbs/doPost.js
+ */
```

Comment:
Looks like the tests are for the functions in "api/v1/helpers/verbs/postUtils.js". Shouldn't you name this file to match that?

Reply:
good point. will change.